strongbox-resources/strongbox-storage-api-resources/src/main/resources/etc/conf/strongbox.yaml

```diff
@@ -4,6 +4,11 @@ configuration:
   revision: ${strongbox.revision}
   baseUrl: http://localhost:48080/
   port: 48080
+  digestAlgorithmSet:
+    - MD5
+    - SHA-1
+    - SHA-256
+    - SHA-512
```

Comment:
This is not the correct place for such configuration. These are only related to Maven repositories, so, obviously, they should only be under the settings of a Maven repository.

Reply:
This is the global digest algorithm set which repositories wil fall back to if they are not overriding the digest algorithms. We're able to configure the global digest algorithm set from strongbox-web-ui settings page.

If my understanding is correct, we currently don't have facility for layout specific global configuration. Please correct me if I'm wrong.

Reply:
I'll check on adding support for layout specific global config

Reply:
As we discussed, I have removed the code for global repository configuration both from strongbox and strongbox-web-ui projects